lib/config.js

```diff
@@ -63,20 +64,54 @@ const getAll = () => cloneDeep(config);
  * @param {*} value Value of a configuration property
  *
  * @returns {*} The value given.
+ *
+ * @deprecated since version 1.3
  */
 const set = (key, value) => {
-  config[key] = value;
+  // NOTE: we cannot use `log` function here as this module is a dependency of logger module.
+  console.warn('config.set is deprecated, please use config.init instead');
```

Comment:
I think this is okay for now, but seeing as the developer-user may not be using a console, we'll want to consider a different approach for v2

Reply:
🙏🏻  

yeah sorry, I think I have a pretty elegant solution using the `emitter` module but it's a breaking change. V2 will be better. 

```
try {
  convertToUsingEmitter()
} catch(err) {
  changeTheTopic()
}
```

lol jk

Reply:
To confirm: There's a ticket for this?

Reply:
https://rewardops.atlassian.net/browse/MX-1212